cves/kernel/CVE-2023-23004.yml

```diff
@@ -409,66 +118,30 @@ lessons:
     note:
   environment_variables:
     applies:
-    note:
+    note: 
   secure_by_default:
     applies:
     note:
   yagni:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: |
+      This principle is likely the closest related to this vulnerability, as it was due to
+      a misunderstanding in a pretty complex system. When dealing with complex data and code, you
+      should be extra careful when making assumptions.
```

Comment:
I would try to avoid language such as "you". Also "likely the closest related" doesn't flow very well when reading it